From ea7852d2dc49e7eac2c1207ab2015c4d3aaaeec0 Mon Sep 17 00:00:00 2001
From: Neena Busireddy <neenareddy.busireddy@nxp.com>
Date: Fri, 19 Sep 2025 13:02:42 -0500
Subject: [PATCH] thirdparty/libbacktrace.cmake: Update cmake_minimum_required
 to 3.5

| CMake Error at thirdparty/libbacktrace.cmake:32 (cmake_minimum_required):
|   Compatibility with CMake < 3.5 has been removed from CMake.
|
|   Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
|   to tell CMake that the project requires at least <min> but has been updated
|   to work with policies introduced by <max> or earlier.

Signed-off-by: Neena Busireddy <neenareddy.busireddy@nxp.com>

Upstream-Status: Pending

---
 thirdparty/libbacktrace.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/thirdparty/libbacktrace.cmake b/thirdparty/libbacktrace.cmake
index c3d81d97..0e7b4ce5 100644
--- a/thirdparty/libbacktrace.cmake
+++ b/thirdparty/libbacktrace.cmake
@@ -29,7 +29,7 @@
 # IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.  */
 
-cmake_minimum_required (VERSION 3.1)
+cmake_minimum_required (VERSION 3.5)
 
 project (libbacktrace)
 
-- 
2.34.1

